### DISCUSSION TITLE
Built-in co-domain projection?

### DISCUSSION THREAD

**Discussion by ego-thales** [GAA-UAM/scikit-fda#539] [Q&A] [answered]
Hi,

Is there a native way to project data in the co-domain to get rid of unwanted dimensions?

For example, suppose I have an `FDataGrid` instance that represents data $\mathbb{R}\rightarrow\mathbb{R}^2$, can I only keep the first dimension of codomain?

I could of course manually tweak the `data_matrix` and instanciate a new `FDataGrid` myself, but I would like to be sure that there is no built-in method for that.

Thanks!

**Comment by vnmabus** [comment, marked as answer]
You can use:
```python
my_fdata.coordinates[0]
```

**Reply by ego-thales** [reply]
Thanks!
